### PR TITLE
Integrate "com.github.breadmoirai.github-release" plugin.

### PR DESCRIPTION
**Pull request by Tapchicoma**
This plugin will automatically sync changelog to github releases page on a new plugin release.

### CODE REVIEW

**Comment by Tapchicoma** [comment]
@JLLeitschuh you will have to add `GITHUB_KEY` to Travis secrets. Key value could be retrieved from "user settings > developer settings" according to https://github.com/BreadMoirai/github-release-gradle-plugin#using-this-plugin

**Comment by JLLeitschuh** [comment]
Okay token has been added.

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ script:
   - ./gradlew ktlintCheck --no-daemon
   - ./gradlew addKtlintCheckGitPreCommitHook --no-daemon && checkbashisms .git/hooks/pre-commit
   - if [ -n "$TRAVIS_TAG" ]; then ./plugin/gradlew -p ./plugin publishPlugins --no-daemon; fi
+  - if [ -n "$TRAVIS_TAG" ]; then ./plugin/gradlew -p ./plugin githubRelease --no-daemon; fi
```
Comment (JLLeitschuh) [inline .travis.yml]: Why not run both tasks in the same build?

**Comment by Tapchicoma** [comment]
> This LGTM. Have you verified the logic on a personal repository?
Nope, never used it before. Will see how it works with next release :sweat_smile:

File: .travis.yml (hunk 1 above)
Comment (Tapchicoma) [reply .travis.yml]: Updated configuration to use same if condition.

**Pull request merged**